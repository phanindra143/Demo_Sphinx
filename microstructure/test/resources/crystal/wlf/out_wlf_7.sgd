*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=    92.5965
  !k_ij(1,2)=     0.6506
  !k_ij(2,2)=    94.0145
  !k_ij(1,3)=    -0.1652
  !k_ij(2,3)=     0.5539
  !k_ij(3,3)=    93.8824
MP,Kxx,1,   92.59647
MP,Kyy,1,   94.01448
MP,Kzz,1,   93.88240
MP,Kxx,2,   94.01448
MP,Kyy,2,   92.59647
MP,Kzz,2,   93.88240
MP,Kxx,3,   94.01448
MP,Kyy,3,   92.59647
MP,Kzz,3,   93.88240
MP,Kxx,4,   92.59647
MP,Kyy,4,   94.01448
MP,Kzz,4,   93.88240
MP,Kxx,5,   92.59647
MP,Kyy,5,   94.01448
MP,Kzz,5,   93.88240
MP,Kxx,6,   94.01448
MP,Kyy,6,   92.59647
MP,Kzz,6,   93.88240
MP,Kxx,7,   94.01448
MP,Kyy,7,   92.59647
MP,Kzz,7,   93.88240
MP,Kxx,8,   92.59647
MP,Kyy,8,   94.01448
MP,Kzz,8,   93.88240
MP,Kxx,9,   93.88240
MP,Kyy,9,   92.59647
MP,Kzz,9,   94.01448
MP,Kxx,10,   92.59647
MP,Kyy,10,   93.88240
MP,Kzz,10,   94.01448
MP,Kxx,11,   92.59647
MP,Kyy,11,   93.88240
MP,Kzz,11,   94.01448
MP,Kxx,12,   93.88240
MP,Kyy,12,   92.59647
MP,Kzz,12,   94.01448
MP,Kxx,13,   93.88240
MP,Kyy,13,   92.59647
MP,Kzz,13,   94.01448
MP,Kxx,14,   92.59647
MP,Kyy,14,   93.88240
MP,Kzz,14,   94.01448
MP,Kxx,15,   92.59647
MP,Kyy,15,   93.88240
MP,Kzz,15,   94.01448
MP,Kxx,16,   93.88240
MP,Kyy,16,   92.59647
MP,Kzz,16,   94.01448
MP,Kxx,17,   94.01448
MP,Kyy,17,   93.88240
MP,Kzz,17,   92.59647
MP,Kxx,18,   93.88240
MP,Kyy,18,   94.01448
MP,Kzz,18,   92.59647
MP,Kxx,19,   93.88240
MP,Kyy,19,   94.01448
MP,Kzz,19,   92.59647
MP,Kxx,20,   94.01448
MP,Kyy,20,   93.88240
MP,Kzz,20,   92.59647
MP,Kxx,21,   94.01448
MP,Kyy,21,   93.88240
MP,Kzz,21,   92.59647
MP,Kxx,22,   93.88240
MP,Kyy,22,   94.01448
MP,Kzz,22,   92.59647
MP,Kxx,23,   93.88240
MP,Kyy,23,   94.01448
MP,Kzz,23,   92.59647
MP,Kxx,24,   94.01448
MP,Kyy,24,   93.88240
MP,Kzz,24,   92.59647
*endif
