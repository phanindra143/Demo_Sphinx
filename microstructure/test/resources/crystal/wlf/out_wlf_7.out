            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\11
             3.00000      -1446948.46062        -17998.71072         -4299.33531
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           138.24123           113.29179
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000          -367.40932      -1468829.47638         14812.99494
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000          -163.35030         15625.00000          -225.33241
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000          -344.81655         12110.00026      -1466771.50487
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000            -8.60194          -246.46257         15625.00000
  /%
